buildscripts/incremental/setup_conda_environment.cmd

```diff
@@ -20,5 +20,5 @@ call activate %CONDA_ENV%
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 @rem Install llvmdev
-%CONDA_INSTALL% -c numba llvmdev="11.*"
+%CONDA_INSTALL% -c numba/label/dev llvmdev="14.*"
```

Comment:
From OOB conversation. The `llvmdev` packages are going to reside in `numba/label/dev` until the stack is more stable to prevent "pollution" of the main `numba` channel.